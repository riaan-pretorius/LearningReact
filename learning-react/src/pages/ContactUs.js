import React, { Component } from 'react';
import { Container, Row, Col } from 'reactstrap'

class ContactUs extends Component {
  render() {
    return (
        <Col>
            <h1>Contact Us</h1>
            <p>Please dont contact us.</p>
        </Col>
    );
  }
}
const styles = {
  
}

export default ContactUs;
