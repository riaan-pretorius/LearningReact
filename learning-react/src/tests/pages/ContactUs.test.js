import React from 'react'
import ContactUs from '../../pages/ContactUs'
import {mount, shallow} from 'enzyme';
import renderer from 'react-test-renderer'

test('ContactUs Test Snapshot', () => {
  const component = renderer.create(
    <ContactUs />
  )
  let test = component.toJSON();
  expect(test).toMatchSnapshot();
});

test('ContactUs Test', () => {
  const page = mount(
    <ContactUs />
  )
  
  let header = page.find('h1')
  expect(header.exists()).toBeTruthy()
  expect(header.text()).toBe("Contact Us")

  let p = page.find('p')
  expect(p.exists()).toBeTruthy()
  expect(p.text()).toBe("Please dont contact us.")
})
