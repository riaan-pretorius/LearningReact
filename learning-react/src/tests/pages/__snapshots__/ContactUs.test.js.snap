// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUs Test Snapshot 1`] = `
<div
  className="col"
>
  <h1>
    Contact Us
  </h1>
  <p>
    Please dont contact us.
  </p>
</div>
`;
